File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int alt = 0;
        int maxAlt = 0;
        
        for(int i: gain){
            alt+=i;
            if (maxAlt<alt){
                maxAlt = alt;
            }
        }
        return maxAlt;
    }
}

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int maxi=0;
        int sum=0;
        for(int i=0;i<gain.size();i++){
            sum=sum +gain[i];
            maxi=max(sum,maxi);
        }
        return maxi;
    }
};
